subsystems/chassis/Chassis.java

```diff
@@ -1,51 +1,44 @@
 package org.usfirst.frc4904.standard.subsystems.chassis;
```

Comment:
update to use `SubsystemBase`, multiple constructors for default command and naming